telemetry_manager.go

```diff
@@ -118,27 +117,17 @@ func (m *TelemetryManager) startCleanUpTimer() {
 			cleanUpInterval*cleanUpIntervalMultiplier) * time.Millisecond)
 
 	go func() {
+	B:
```

Comment:
Can we use a more descriptive label here?

Reply:
Modified to CleanUpTimerLabel